indexer_app/handler.py

```
@@ -1,73 +1,228 @@
     handle_transfer_payout,
+    handle_update_campaign,
+    handle_update_default_human_threshold,
+    process_rounds_deposit_event,
+    process_vote_event,
+    update_application,
+    update_round_payout,
 )
 
 
 async def handle_streamer_message(streamer_message: near_primitives.StreamerMessage):
+    start_time = time.time()
```

Ruff (0.12.2)

Local variable `start_time` is assigned to but never used

Remove assignment to unused variable `start_time`

(F841)